From 3ebf079cbceb38151fbe53bbbf2c948e7efe0dc9 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Tue, 17 May 2016 10:45:08 +0800
Subject: [PATCH 1070/5242] MLK-12781 ARM: dts: imx7d-12x12-lpddr3-arm2: add
 pinctrl for usb vbus

commit  f257c0e976b0c151bb2cdbed8e0f81d807d1f3e3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add pinctrls for usbotg1 and usbotg2 vbus control. This missing keeps
the vbus enable pin is high after power up, so vbus is on and otg port
will not enter suspend in device mode, as active usb port has high
bus freq requested, this prevents system enter low bus freq.

Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index eb87225..0e2c7cb 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -951,6 +951,8 @@
 
 &usbotg1 {
 	vbus-supply = <&reg_usb_otg1_vbus>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_usbotg1_vbus>;
 	srp-disable;
 	hnp-disable;
 	adp-disable;
@@ -959,6 +961,8 @@
 
 &usbotg2 {
 	vbus-supply = <&reg_usb_otg2_vbus>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_usbotg2_vbus>;
 	srp-disable;
 	hnp-disable;
 	adp-disable;
-- 
1.7.9.5

